<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calcolatrice</class>
 <widget class="QDialog" name="Calcolatrice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>830</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Calcolatrice</string>
  </property>
  <widget class="QTabWidget" name="Grafici">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>741</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-left-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);


</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabTriangolo">
    <attribute name="title">
     <string>Triangolo</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="quattroLati">
    <attribute name="title">
     <string>4 Lati</string>
    </attribute>
    <widget class="QTabWidget" name="graficiQuad">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>741</width>
       <height>516</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-top-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);

QTabBar::tab  {
	border-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabQuadrilatero">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <attribute name="title">
       <string>Quadrilatero</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabRettangolo">
      <attribute name="title">
       <string>Rettangolo</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabQuadrato">
      <attribute name="title">
       <string>Quadrato</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="Risultato">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(202, 202, 202);

border: 0.5px solid black;

border-radius: 2px;</string>
   </property>
   <property name="text">
    <string>Risultato</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="inserisciFigura">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Inserisci una nuova figura</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Inserisci Figura</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Icone/inserisci.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labRisultati">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>610</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(100, 100, 100);

padding-left: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Somma">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="accessibleName">
    <string comment="Somma campi info" extracomment="Somma campi info"/>
   </property>
   <property name="text">
    <string>Somma +</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="eliminaVertice">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>660</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina Vertice</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="inserisciVertice">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>660</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci Vertice</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="eliminaFigura">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Elimina Figura</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Icone/delete.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="eliminaTutto">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Elimina Tutto</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Icone/deleteAll.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Esci">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string> Esci</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Icone/power-button.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="modificaInfo">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>700</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifica Info</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Sottrazione">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>700</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="accessibleName">
    <string comment="Somma campi info" extracomment="Somma campi info"/>
   </property>
   <property name="text">
    <string>Sottrazione -</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
